_pages/install/on_windows.md

```diff
@@ -2,15 +2,27 @@
 subtitle: On Windows (Preview)
 ---
 
-To easily install the Crystal Preview on Windows you can use [Scoop](https://scoop.sh/). 
+To easily install Crystal on Windows you can use the [Scoop](https://scoop.sh/) preview package. 
 **Be aware of the fact that Crystal on Windows is** [**not yet complete**](https://github.com/crystal-lang/crystal/issues/5430).
 
-Before you start the installation, you must ensure that the [required packages](https://github.com/neatorobito/scoop-crystal#requirements) are installed.
+## Requirements
+* Developer mode enabled in Settings
+* [Scoop](https://scoop.sh/)
+* x64 Native Tools Command Prompt in Windows Terminal
```

Comment:
This is a bit confusing. It's technically a prerequisite for `crystal`, but you can also install it via scoop.
I think we should either directly mention here that it's included in the installation instructions, or not name it here at all, only in the installation section.

Reply:
fair point, it should be consistent with the `git` requirement

Reply:
Agreed, I've pushed new updates that address this.